src/app/mytools/my-tool/my-tool.component.html

```diff
@@ -98,6 +98,17 @@
                 [groupEntriesObject]="groupEntriesObject$ | async"
                 [refreshMessage]="isRefreshing$ | async"
               ></app-sidebar-accordion>
+              <br />
+              <small
+                >Don't see an organization?
+                <a
+                  href="https://github.com/settings/connections/applications/7ad54aa857c6503013ea"
```

Comment:
I didn't know we had such a link. :)

No need to change, could be a followup ticket, could be overkill and not worth it since this will rarely come into play, but ideally the installation id would come from the config.json, so that it would work on staging and qa. And also if it ever changes -- I don't know if they're immutable or not, couldn't tell from a quick Google search.

Reply:
This is the oauth app's client ID according to this https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/authorizing-oauth-apps#directing-users-to-review-their-access

We have access to this value here https://github.com/dockstore/dockstore-ui2/blob/e221a20669591eb9b25ebc13267eab7f2a03598c/src/app/shared/dockstore.model.ts#L50

Seems like a simple fix so I'll do it in this PR and re-request review after testing it :slightly_smiling_face: